che-incubator/che-code
# Update dependencies

### What does this PR do?
Upgrade Che-Code dependencies.

### What issues does this PR fix?

Within last alignment Che-Code with upstream I faced a problem related to different versions of che-code dependencies.
I noticed that some versions are pretty old.
So within current PR I'm going to upgrade some dependencies.
Typescript version should be updated within https://github.com/eclipse-che/che/issues/23083 after resolving https://github.com/eclipse-che/che/issues/23082.

### How to test this PR?
I tested the following `che` functionality:
- Stop workspace
- Restart workspace from local devfile
- Run a devfile task
- Git clone (che-github-authentication + che-api extensions are used)
- Device Authentication
- Open a URL from the `Endpoints` panel
- checked that `Workspace resources` are displayed (che-resource-monitor is used)
- che terminal creation
- suggestion from the che-port extension is displayed when a sample project is started
- 
### Does this PR contain changes that override default upstream Code-OSS behavior?

Signed-off-by: Roman Nikitenko <rnikiten@redhat.com>

rh-pre-commit.version: 2.2.0
rh-pre-commit.check-secrets: ENABLED
